aws-codeartifact-repository/src/main/java/software/amazon/codeartifact/repository/Translator.java

```diff
@@ -72,7 +73,12 @@ static CreateRepositoryRequest translateToCreateRequest(final ResourceModel mode
    * @return list of UpstreamRepository
    */
   static List<UpstreamRepository> translateToUpstreamList(final ResourceModel model) {
-    return streamOfOrEmpty(model.getUpstreams())
+    if (model.getUpstreams() == null) {
+      return null;
```

Comment:
do we want null? or is an empty list better? Not sure how either is handled via the sdk, so null might be correct

Reply:
In general I'd avoid null collections and use empty ones instead. Do we know that the generated ResourceModel does? Does it even return null for `getUpstreams()` or does it always return empty?

Reply:
The generate resource model will be null if there property is not specified. I'll update this return an empty list instead